packages/info/tsconfig.json

```diff
@@ -1,3 +1,3 @@
-{
-  "extends": "../../tsconfig.packages.json"
-}
+{
```

Comment:
Again these changes... could please make sure that are not touched again? It's not the first time

Reply:
I will try again. Can you guide me here?

Reply:
We have a `.gitattributes` file. Could change it to `* text eol=lf` and try again?